src/Paket.Core/Environment.fs

```diff
@@ -63,6 +63,18 @@ module PaketEnv =
             |> Seq.tryFind File.Exists
             |> Option.map (fun f -> DirectoryInfo(Path.GetDirectoryName(f)))
 
+    let ensureNotExists (directory : DirectoryInfo) =
+        match fromRootDirectory directory with
```

Comment:
I'm ok with that, but then maybe move back the ``ensureNotExists`` to use for nugetConvert module only